src/__test__/components/Collapsible/Collapsible.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import Collapsible from "@components/Collapsible";
+import { render, screen } from "@testing-library/react";
+import '@testing-library/jest-dom'
+
+describe('Collapsible', () => {
+    test('Checking collapsible title', () => {
+        render(<Collapsible title={'Teste'} />);
+        const button = screen.getByRole('button', { name: 'Teste' });
+
+        expect(button).not.toBeDisabled();
+        expect(button).toHaveTextContent('Teste');
```

Comment:
🔪 

This line is not needed, since the `getByRole` already verified if the button exist with the text `Test`